### PR TITLE
Three seconds is better

### PR DESCRIPTION
**Pull request by Viktor-Yegay**
Studies have shown that 3 seconds is a far better LED delay than 1 second.

http://studies.example.com/optimal-led-delays.html

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(3000);               // wait for a second
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
-  delay(2000);               // wait for a second
+  delay(3000);               // wait for a second
```
Comment (Viktor-Yegay-It) [inline blink.ino]: I believe it would be better if the light was off 4 seconds and on for just 3.

**Comment by Viktor-Yegay-It** [comment]
If you make that change, I'll be happy to merge this.

**Comment by Viktor-Yegay** [comment]
I change it to 4 seconds and also removed some trailing whitespace that I found. Anything else you would like me to do?

**Comment by Viktor-Yegay** [comment]
Этот PR заменяет #1 в качестве ребазированной ветки. Вы также должны увидеть viktor-yegay#1 и, конечно, schacon/kidgloves#2 Хотя ничто не сравнится с https://github.com/schacon/kidgloves/issues/1.

- [x] Write the code
- [ ] Write all the tests
- [ ] Document the code

**Comment by Viktor-Yegay** [comment]
> If you make that change, I'll be happy to merge this.

sdfsf

**Pull request merged**